fw/cache.c

```diff
@@ -3292,11 +3292,12 @@ tfw_cache_start(void)
 static void
 tfw_cache_stop(void)
 {
+	TfwGlobal *g_vhost = tfw_vhost_get_global();
 	int i;
 
 	if (tfw_runstate_is_reconfig())
 		return;
-	if (!cache_cfg.cache)
+	if (!(cache_cfg.cache || g_vhost->cache_purge))
```

Comment:
This condition in `tfw_cache_start()` looks strange, why do we need to initialize cache if cache disabled, but purging is enabled? For instance in this case we initialize the cache: `cache 0; cache_purge invalidate;`, looks wrong.

Reply:
Yes, agree - we should not allow configuration of cache purging if we don't have cache at all! @EvgeniiMekhanik could you please fix this in `tfw_cache_start()` and add a test that this configuration isn't allowed?